src/pages/default/connections/ReviewConnection.tsx

```diff
@@ -0,0 +1,89 @@
+import { Button } from "@/components/Button";
+import { BadgeNames, Item } from "@/components/Item";
+import { LabelValueItem } from "@/components/LabelValue";
+import { ColorTheme } from "@/theme/colors";
+import { FlexLayouts, Layouts } from "@/theme/layouts";
+import { Typography } from "@/theme/typography";
+import { formatDID, formatDate } from "@/util/formatters";
+import React from "react";
+import { View, SafeAreaView, Text } from "react-native";
+
+const ReviewConnectionScreen = ({ route }) => {
+  //TODO Sub out with real connectionSets
+  const { connection, profile, dateCreated } = route.params.connectionSet || {
+    connection: {
+      name: "Dignal",
+      icon: "comment-discussion",
+    },
+    profile: {
+      name: "Social profile",
+      displayName: "Alex Aardvark",
+      icon: "hash",
+      id: "did:ion:123456789012345678901234567890",
+    },
+    dateCreated: "Sat July 01 2023",
+  };
+  return (
+    <SafeAreaView>
+      <View style={Layouts.container}>
+        <View style={Layouts.row}>
+          <Item
+            heading={profile.name}
+            subtitle={profile.displayName}
+            body={formatDID(profile.id)}
+            iconName={profile.icon}
+            badgeName={BadgeNames.PROFILE}
+            headingSize="heading3"
+          />
+          <Item
+            heading={connection.name}
+            iconName={connection.icon}
+            badgeName={BadgeNames.CONNECTION}
+            headingSize="heading3"
+          />
+        </View>
+        <View style={Layouts.row}>
+          <Text style={Typography.body4}>
+            <Text style={Typography.body2}>{connection.name}</Text> is connected
+            to Professional profile and has access to certain info.
+          </Text>
+        </View>
+        <View>
+          <LabelValueItem
+            label="Connection made"
+            value={formatDate(dateCreated)}
+          />
+        </View>
+        <View style={Layouts.row}>
+          <Text style={Typography.body4}>
+            <Text style={Typography.body2}>{connection.name}</Text> can
+          </Text>
+          <Text style={Typography.body4}>
+            &bull; see and edit your profile info
+          </Text>
+          <Text style={Typography.body4}>
+            &bull; see and edit your contacts
+          </Text>
+          <Text style={Typography.body4}>&bull; see and edit your chats</Text>
+        </View>
+        <View style={FlexLayouts.column}>
+          <Button kind="destructive">
+            Disconnect <Text style={Typography.body2}>{connection.name}</Text>{" "}
+            from <Text style={Typography.body2}>{profile.name}</Text>
+          </Button>
+          <Text
+            style={[
```

Comment:
This one feels random. I feel like textCenter shouldn't exist. We should primarily be relying on constants. If we can to lean on composition then I think Tailwind is the way to go